### PR TITLE
Add pt::text()

### PR DESCRIPTION
**Pull request by j33ty**
1. Add options to build the test suite:
    - Outputs to suite.ndjson: `yarn build`
    - Outputs to stdout: `yarn build --stdout`
    - Outputs to custom.ndjson: `yarn build --out=custom.ndjson`
    - Only build for files which matched the pattern: `yarn build --pattern='**/misc.yml'`
2. Add tests for `pt::text()`.

### CODE REVIEW

File: test/extensions/pt/text.yml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+          "_type": "span",
+          "marks": [],
+          "text": "This is one of her most famous portraits:"
+        }],
+        "markDefs": []
+      }
+tests:
+  - query: |
+      sanity::blockText(body)
+    result: "Heading1"
```
Comment (judofyr) [inline test/extensions/pt/text.yml:262]: This doesn't look complete?

**Review by judofyr — COMMENTED** [review]
Overall changes look good, but there doesn't seem to be any actual tests here.

File: test/extensions/pt/text.yml [hunk 2]
```diff
@@ -0,0 +1,262 @@
+name: "PT Text"
+
+features:
+  - namespaces
+  - experimentalFunctions
+
+documents:
+  - _id: "block-array-invalid-blocks-lenient"
+    _type: "test"
+    body: |
```
Comment (judofyr) [inline test/extensions/pt/text.yml:10]: Portable Text is typically not stored as a string, but as "real" arrays/objects.

Comment (j33ty) [reply test/extensions/pt/text.yml]: Fixed.

File: test/extensions/pt/text.yml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+          },
+            {
+              "_type": "span",
+              "marks": ["emphasis"],
+              "text": "Annie Leibovitz"
+            }
+          ]
+        }
+      ]
+tests:
```
Comment (atombender) [inline test/extensions/pt/text.yml:104]: Just one test but a bunch of documents? Shouldn't we test more of these?

Comment (atombender) [reply test/extensions/pt/text.yml:104]: Never mind, I misread the query!

**Review by atombender — APPROVED** [review]

**Review by judofyr — APPROVED** [review]

**Pull request merged**